pyqtgraph/tests/test_makeARGB.py

```diff
@@ -4290,9 +4290,11 @@ def assert_correct(data, key, levels, lut, scale, use_rgba):
     _do_something_for_every_combo(assert_correct)
 
 
+setConfigOption("useCupy", True)
```

Comment:
The same Python process is used to run all the tests, including those tests outside of this file. So any config option that gets modified needs to be restored back to its old value after.

Reply:
I'm thinking we should make a `restore_useCuPy` fixture that will automatically revert `useCuPy` after a test that uses it is finished.  I know we could manually add a few lines to the test, that isn't terribly complicated, but it's just going to be one more thing you have to remember should you add/review a test; a fixture might be easier to understand what it's doing.